docs/using-actuated.md

```diff
@@ -0,0 +1,39 @@
+# Using actuated
+
+[Actuated](https://actuated.dev/) is available as a GitHub App which can be
+enabled on repositories, and allows GH Actions to execute on runners hosted by
+the CNCF. [See the announcement](https://actuated.dev/blog/arm-ci-cncf-ampere)
+
+## Installation
+
+Before it can be used, the App has to be enabled on the repository, through a
+maintenance request.  [See an
+example](https://github.com/open-telemetry/community/issues/1954).
+
+## Usage
+
+Using actuated on a GitHub Actions job consists in changing the `runs-on`
+section to be `actuated-*` (to run jobs on x86), or `actuated-arm64-*`, with
+the required CPUs and RAM in the name.
```

Comment:
Is there a place where I can look up the available list of `actuated-*` ?

Reply:
They mention on their docs that `actuated-*` and `actuated-arm64-*` are available, but I'm not seeing any matrix that would give the list of all available values.
https://docs.actuated.dev/

Reply:
Ok, then we might need to follow up with actuated team here? Let's put this in a follow up issue, such that it is non-blocking

Reply:
I have pinged them on their slack.

Reply:
So, their answer is actually that we could have anything we want. But CNCF orgs only have arm64 enabled (it's all on bare metal on equinix as far as I understand). 
I'll remove the mention of non-arm64.
https://actuated.dev/blog/arm-ci-cncf-ampere

Reply:
thanks for the follow up! what can we put there instead of the `*`? In the blog post I see

runs-on: actuated-arm64

and

runs-on: actuated-arm64-8cpu-16gb

and

actuated-arm64-8cpu-32gb.

So it's something like CPU number + RAM size?

Reply:
Yes. As far as I can see, that's all that can be specified. I suppose they use cgroup (maybe through k8) limits, so setting a value higher than what is available may never start the runner.

@atoulme may know more about what machine we have available?

Reply:
as per @alexellis comment below https://docs.actuated.dev/examples/custom-vm-size/ is what we should be referencing then